org.darktable.Darktable.json

```diff
@@ -78,6 +83,24 @@
                 }
             ]
         },
+        {
+            "name": "lensfun-db",
+            "buildsystem": "simple",
+            "build-commands": [
+                "tar xf version_2.tar.bz2",
+                "rm version_2.tar.bz2",
+                "date +%s > timestamp.txt"
+            ],
+            "post-install": [
+                /* Copy over the latest lens definition files */
+                "cp -r /run/build/lensfun-db/* /app/share/lensfun/version_2/"
+            ],
+            "sources": [{
+                "type": "file",
```

Comment:
Why not make it `type: archive` and save the manual unpacking?

Reply:
I had it as an archive at first, but flatpak builder did not want to unpack the archive.

Reply:
Open the archive and notice that the file are not contained in a directory! That's why you have to add `strip-components: 0`.